var/spack/repos/builtin/packages/sundials/package.py

```diff
@@ -87,7 +88,7 @@ class Sundials(CMakePackage, CudaPackage, ROCmPackage):
     # Logging
     variant(
         "logging-level",
-        default="0",
+        default="2",
```

Comment:
Is there a way to make the default version dependent so the previous default is retained in older versions?

Reply:
Some other updates below:
* `logging-mpi` variant, `when` can be updated to `when="@6.2.0:6.7.0"`
* `generic-math` variant, add `when="@:6.7.0"`

Reply:
Updated `logging-mpi` and `generic-math`. There does not currently seem to be a way to have the default depend on the version.